sycl/doc/extensions/experimental/sycl_ext_oneapi_device_global.asciidoc

```diff
@@ -305,6 +312,20 @@ The storage on each device for `T` is zero-initialized.
 
 `T` must be trivially default constructable and trivially destructible.
 
+// --- ROW BREAK ---
+a|
+[source,c++]
+----
+template <typename... Args>
+consteval explicit device_global(Args&&... args);
+----
+|
+Constructs a `device_global` object, and implicit storage for `T` in the global address space on each device that may access it.
+
+The storage on each device for `T` is initialized with the contents of the arguments.
```

Comment:
I don't think this is true.  Consider this case:

```
struct foo {
  int sum;
  int difference;
  consteval foo(int a, int b) {
    sum = a + b;
    difference = a - b;
  }
};

device_global<foo> f{3, 4};
```

In this case f is initialized not to the contents of the arguments but by calling foo's constructor.  This wording makes it sound like we always use aggregate initialization rather than calling a constructor.  Perhaps we could say that this initializes the underlying allocation by calling T's constructor with args.

Reply:
Should we say something like:

> The object of type `T` is initialized from the `args` parameter pack using list initialization as defined in the C++ specification.

I think the rules for [list initialization](https://en.cppreference.com/w/cpp/language/list_initialization) define which cases use aggregate initialization vs. calling a constructor.

Reply:
Updated!